### PR TITLE
Fixed issue #312

### PR DESCRIPTION
**Pull request by Varun270**
### Your checklist for this pull request

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your main!
- [ ] Make sure you are making a pull request against the **development branch** (left side). Also you should start *your branch* off *our development*.
- [ ] Check the commit's or even all commits' message styles matches our requested structure.
- [ ] I have added necessary documentation (if appropriate)
- [ ✓]  It's small enough so they can be easily reviewed.

- [ ] I have CONSIDERED or ADDED a unit test if the PR resolves an issue.

## Types of changes
What types of changes does your code introduce to Robocomp?  

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code style update (formatting, renaming)
- [✓ ] Documentation Update
- [ ] Build related changes
- [ ] Other (please describe):

### Does this introduce a breaking change?

- [ ] Yes
- [ ✓] No

### Does this close any currently open issues?
Issue ID: [#312  ]

### Description
Please describe your pull request.
I have fixed the typos in docker documentation.

===  
**Thank you!**

### CODE REVIEW

File: docker/Readme.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ We have also developed a tool on top of docker that you could be interested in:
 
 
 ### Other Dockerfiles related with Robocomp
-This other Dockerfiles, dockercompose files and related images in docker hub could be of your interest if you are working with these projects.
+These other Dockerfiles, docker-compose files,and related images in the docker hub could be of your interest if you are working with these projects.
```
Comment (orensbruli) [inline docker/Readme.md:40]:
```suggestion
These other Dockerfiles, docker-compose files, and related images in the docker hub could be of your interest if you are working with these projects.
```

**Comment by orensbruli** [comment]
Thank you for your contribution!
This issue has conflicts with previous modifications of the same file. Can you update your fork  and check again? Thank you!

**Comment by Varun270** [comment]
@orensbruli  I am generating another pull request with suggested changes . Thanks for telling.

**Comment by Varun270** [comment]
@orensbruli  I am generating another pull request with suggested changes . Thanks for telling.

File: docker/Readme.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ We have also developed a tool on top of docker that you could be interested in:
 
 
 ### Other Dockerfiles related with Robocomp
-This other Dockerfiles, dockercompose files and related images in docker hub could be of your interest if you are working with these projects.
+These other Dockerfiles, docker-compose files, and related images in the docker hub could be of your interest if you are working with these projects.
```
Comment (Varun270) [inline docker/Readme.md:40]: @orensbruli  I have one doubt related to the suggested changes that , I didn't particularly understood what change you want me to do in  "if" word . According to me it's grammatically correct.

**Comment by Varun270** [comment]
@orensbruli  why did you closed this pulled request?

File: docker/Readme.md (hunk 1 above)
Comment (orensbruli) [reply docker/Readme.md]: I don't remember to have changed anything to the "if" word, and it doesn't look like it's changing. I don't know why github is showing this as a change.

**Comment by orensbruli** [comment]
> @orensbruli why did you closed this pulled request?

Becuase of the unresolved conflict with the original file. Can you update your fork and retry?

**Comment by Varun270** [comment]
@orensbruli  I did the changes suggested by you but now I am having an issue in pushing those changes. The error message in gi t that I got is "Updates were rejected because the tip of your current branch is behind". Since I am a beginner I am not familiar with resolving conflicts.

**Comment by orensbruli** [comment]
> @orensbruli I did the changes suggested by you but now I am having an issue in pushing those changes. The error message in gi t that I got is "Updates were rejected because the tip of your current branch is behind". Since I am a beginner I am not familiar with resolving conflicts.

Yes, that is what I mean when I said I closed your PR because of a conflict with the current version. Another contributor made changes to this file after you forked. You can see it here: https://github.com/robocomp/robocomp/pull/323/conflicts

We have two options now:
1.- You can locally resolve those conflicts by updating your fork to the current state and fixing problems:
https://medium.com/@sahoosunilkumar/how-to-update-a-fork-in-git-95a7daadc14e
2.- We can consider this PR as deprecated, close it and you can check the last version of the file and decide if you want to do a new PR.

In my opinion, the first version is the one that would teach you more about how to work with github.

**Comment by Varun270** [comment]
@orensbruli  I am creating a new pull request for this issue after resolving the conflicts and thanks for the advice, I will probably go with your 1st opinion.

**Comment by Varun270** [comment]
git1
This is the error message that I am receiving while trying to use git pull.

**Comment by orensbruli** [comment]
This is not an error. It's just telling you that you need to merge a conflict or discard your changes.

**Pull request closed without merging**